proc/groupby.go

```diff
@@ -335,68 +339,71 @@ func typeMatch(typeCol []zeek.TypedEncoding, rowkeys []zeek.TypedEncoding) bool
 // recordsForTable returns a slice of records with one record per table entry in a
 // deterministic but undefined order.
 func (g *GroupByAggregator) recordsForTable(table map[string]*GroupByRow) []*zson.Record {
+
+	// XXX get rid of this
+	oldtable := table
+	table = make(map[string]*GroupByRow)
+	for key, val := range oldtable {
+		zv := zval.Encoding(key[4:])
+		oldkey := zv.String()
+		table[oldkey] = val
+	}
+	// ^^^ get rid of this
+
 	var keys []string
 	for k := range table {
 		keys = append(keys, k)
 	}
-	sort.Strings(keys)
-	var recs []*zson.Record
+	// XXX get rid of [4:]
```

Comment:
Is this XXX comment a leftover from an earlier rev?

Reply:
Yes, I was trying to pass old baseline